src/main/webapp/app/entities/statistics.model.ts

```diff
@@ -0,0 +1,6 @@
+export enum SpanType {
+    DAY = 'DAY',
```

Comment:
Just fyi but you can also define enums like:

```
enum SpanType {
   Day,
   Week, 
   Month, 
   Year
```

You don't need to always write the string representation.

Reply:
Good to know, however i would like to keep it that way in order to keep it consistent, as we use this declaration all the time.